waiter/src/waiter/util/client_tools.clj

```diff
@@ -288,11 +288,12 @@
 
 (defn make-request
   ([waiter-url path &
-    {:keys [body client cookies content-type disable-auth form-params headers
+    {:keys [body client cookies content-type disable-auth follow-redirects? form-params headers
```

Comment:
Is `follow-redirects?` used?

Reply:
not yet - was trying to get the http call to work

Reply:
If not used, we should undo this change.